bionic/code_hasher.py

```diff
@@ -219,6 +219,19 @@ def _update_hash(self, hash_accumulator, obj, code_context):
                 obj_bytes=self._check_and_hash(obj.value, code_context),
             )
 
+        elif isinstance(obj, (property, types.DynamicClassAttribute)):
```

Comment:
When would we encounter a `types.DynamicClassAttribute`? Is there a test case for this?

Reply:
`Enum`s use it and we hash `TypePrefix` that tests for it. But related to [this comment](https://github.com/square/bionic/pull/329#discussion_r571264494), I'll see how hard it is to create a `DynamicClassAttribute` and test it directly (like property tests).